newIDE/app/src/UI/SoundPlayer/PlayButton.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+
+import * as React from 'react';
+import IconButton from '@material-ui/core/IconButton';
+import Play from '../CustomSvgIcons/Play';
+import Pause from '../CustomSvgIcons/Pause';
+import { makeStyles } from '@material-ui/styles';
+import GDevelopThemeContext from '../Theme/GDevelopThemeContext';
+
+type Props = {|
+  primary?: boolean,
+  isPlaying: boolean,
+  onClick: () => void,
+|};
+
+const PlayButton = (props: Props) => {
+  const gdevelopTheme = React.useContext(GDevelopThemeContext);
+
+  const classes = makeStyles({
```

Comment:
is this a case where we should prioritize a .css file instead of makeStyles?

Reply:
I'm not sure because I'm using a lot of Material UI features (ripple + event listeners).